features/step_definitions/smokey_steps.rb

```diff
@@ -57,6 +57,28 @@
   visit_path path_or_url
 end
 
+When /^I visit the "([^"]+)" finder with keywords (.*)$/ do |finder, keywords|
+  path = "/search/#{finder}?keywords=#{keywords}"
+  visit_path path
+end
+
+And /^There should be no alert$/ do
+  expect {
+    page.driver.browser.switch_to.alert.accept
+  }.to raise_error(Selenium::WebDriver::Error::NoSuchAlertError)
```

Comment:
This seems to try to close an alert and then raise an error if there isn't an alert to close. Is that right?

Is there another way to check if an alert has been created?

Reply:
Not that I could find.